subsys/net/ip/ipv4_fragment.c

```diff
@@ -467,7 +467,7 @@ static int send_ipv4_fragment(struct net_pkt *pkt, uint16_t rand_id, uint16_t fi
 
```

Comment:
Please change the commit subject to `net: ipv4: Fix tx_pkts slab leak in send_ipv4_fragment`

Reply:
updated

Reply:
Not sure why you try to keep the `module:` prefix in the commit subject, just remove it. You also need to change the commit in you tree and then force push the PR to update the commit message.

Reply:
remove "modules" title